### PR TITLE
Basic User Management

### LINKED ISSUE #5: User management

### PR DESCRIPTION
**Pull request by scottrobertson**
UI + Rake task

```
rake users:create[email@user.com]
rake users:create[email@user.com, a_password]
```

Closes #5

### CODE REVIEW

File: app/controllers/users_controller.rb [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+      redirect_to users_path
+    else
+      render :new
+    end
+  end
+
+  def destroy
+    @user = User.find(params[:id])
+    @user.destroy
+    redirect_to :back
```
Comment (marcqualie) [inline app/controllers/users_controller.rb:24]: I could totally just go and delete your user :trollface:

File: app/controllers/users_controller.rb [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+    @users = User.order(created_at: :desc)
+  end
+
+  def create
+    @user = User.new(create_params)
+    if @user.save
+      flash[:info] = "#{@user.email} has been created with the password of #{create_params[:password]}"
+      redirect_to users_path
+    else
+      render :new
```
Comment (marcqualie) [inline app/controllers/users_controller.rb:17]: No flash message if there's an error? I have a feeling devise doe something clever with the model.errors here, but can't remember

File: lib/tasks/users.rake [hunk 3]
```diff
@@ -0,0 +1,16 @@
+namespace :users do
+  task :create, [:email, :password] => :environment do |t, args|
+    unless args[:email].present?
+      puts 'Email is required'
+      next
+    end
+
+    unless args[:password].present?
+      puts 'Password is required'
```
Comment (marcqualie) [inline lib/tasks/users.rake:9]: You can use `abort('Password is required')` in Rake tasks which outputs the message and returns an exit code of 1

File: app/controllers/users_controller.rb (hunk 1 above)
Comment (marcqualie) [reply app/controllers/users_controller.rb:24]: Based on the UI I guess this is intended. Everyone is super admin for this version?

Comment (scottrobertson) [reply app/controllers/users_controller.rb:24]: There are no permissions etc. "Basic" :D

File: lib/tasks/users.rake (hunk 3 above)
Comment (scottrobertson) [reply lib/tasks/users.rake:9]: Oh nice. Will do that.

File: app/controllers/users_controller.rb (hunk 2 above)
Comment (scottrobertson) [reply app/controllers/users_controller.rb:17]: It's actually just a "simple_form" form, which handles all the error output automatically.

**Pull request merged**